spec/requests/api/graphql_controller_spec.rb

```diff
@@ -18,7 +18,7 @@ def result_source_tenant(response_body)
       post("/api/v1.0/graphql", :headers => headers, :params => graphql_source_query)
 
       expect(response.status).to eq(200)
-      expect(result_source_tenant(response.body)).to match_array([tenant.external_tenant])
+      # expect(result_source_tenant(response.body)).to match_array([tenant.external_tenant])
```

Comment:
Commented out for now. Need to decide if this test is still needed now that tenant is no longer returned

Reply:
@gtanzillo you want to delete this line?